db/migrate/20210615232805_add_in_state_tuition_information_to_institutions_archives.rb

```diff
@@ -0,0 +1,5 @@
+class AddInStateTuitionInformationToInstitutionsArchives < ActiveRecord::Migration[6.0]
+  def change
+    add_column :institutions_archives, :in_state_tuition_information, :string
```

Comment:
Can we just add this line under the line for institutions? I think they go together